### PR TITLE
Adding changlog for ADB 2.2.2 release

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty lmohanty@redhat.com

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v2.2.2 July 18, 2016
```
Comment (brgnepal) [inline CHANGELOG.md]: Looks like it is four letter month name against the three letter word name in VSM.

Comment (LalatenduMohanty) [inline CHANGELOG.md]: @budhrg is it an issue in your opinion?

**Comment by brgnepal** [comment]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (LalatenduMohanty) [inline CHANGELOG.md]: @budhrg fixed it.

Comment (brgnepal) [inline CHANGELOG.md]: @LalatenduMohanty No. Just that I noticed it and found there should be unifying policy across ADB ecosystem projects. :smile:

**Comment by praveenkumar** [comment]
LGTM

**Pull request merged**